[RM Ch03] Add redfish as Hardware Infrastructure Manager option (#3413)
anuket-project/anuket-specifications: doc/ref_model/chapters/chapter03.rst

- **Additional Management Functions**: include software and configuration life cycle management, identity management,
  access management, policy management (e.g. to enforce security policies), etc.

The Redfish® Standard based Hardware Infrastructure Manager 
-----------------------------------------------------------

> This section proposes a Redfish based hardware infrastructure manager for the Anuket project. ODIM software architecture is an example of an open source project implementing Redfish standard. 

Reviewer: ```suggestion
This section proposes a Redfish based hardware infrastructure manager for the Anuket project. Open Distributed Infrastructure Management (ODIM) is an open source software platform that delivers means of distributed physical infrastructure lifecycle management based on the industry standard DMTF Redfish API and Model specification.
```

> Chapter 9 (:ref:`chapters/chapter09:Configuration and Lifecycle Management`) of this reference model specifies redfish standard for managing Infrastructure Hardware. The GSMA's "Cloud Infrastructure Reference Model (NG.126)" also specifies Redfish as the standard interface that should be 

Reviewer: ```suggestion
Chapter 9 (:ref:`chapters/chapter09:Configuration and Lifecycle Management`) of this reference model specifies Redfish standard for managing Infrastructure Hardware. The GSMA's "Cloud Infrastructure Reference Model (NG.126)" also specifies Redfish as the standard interface that should be 
```

-	**Topology**: The Redfish schema contains a cable model

-	**Equipment**: The Redfish schema contains a physical model and a functional model. The physical model expresses the chassis and containers and interconnects between them and environmental control functions.  The functional model expresses the logical aspects and includes configuration and  firmware status.   Both the physical and functional model have their own health/fault status information

-	**Resource Allocation and Composition**: The Redfish schema has a composition model through which a client can compose a logical resource by allocating physical resources. `Redfish Composition Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2050_1.2.0_1.pdf>`__

-	**Underlay Network Resources Manager**: The Redfish schema has models for fabrics, switches and SmartNICs.

-	**Monitoring**: The Redfish schema contains event model for the client to receive hardware events and telemetry model for collecting information across the entire model (physical and functional) `Redfish Telemetry Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf>`__

> -	**Additional Management Functions**: The Redfish schema has models for access management and identity management. Redfish has published work-in-progress extension for support policy management and OS/VM management `Policy Model Proposal, Redfish for Operating Systems, Applications and Containers <https://www.dmtf.org/documents/redfish-spmf/redfish-operating-systems-applications-and-containers-05wip>`__

Reviewer 2: this link isn't publicly readable (access - denied).
Would you mind replacing it or worse making an linkcheck exception in setup.py

> How redfish fits into the ETSI NFV Networking Reference Model

Reviewer: ```suggestion
How Redfish fits into the ETSI NFV Networking Reference Model
```
